app/app.go

```diff
@@ -661,7 +666,7 @@ func New(
 		icaModule.InitModule(ctx, controllerParams, hostParams)
 
 		ctx.Logger().Info("start to run module migrations...")
-		return fromVM, nil
+		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
```

Comment:
@ankurdotb This is the actual way to call registered migration handlers for a module consensus version.